frontend/src/app/components/EmailForm.js

```diff
@@ -0,0 +1,92 @@
+import classnames from 'classnames';
+import React from 'react';
+import { Field, reduxForm } from 'redux-form';
+
+
+class EmailForm extends React.Component {
+  makeRevealedClassNames() {
+    const { data } = this.props;
+    return classnames('revealed-field', {
+      reveal: data && data.email
+    });
+  }
+
+  renderEmailFieldComponent(field) {
+    return (
+      <input {...field.input} placeholder='Your email here' type='email'
+             data-l10n-id='emailFormEmailPlaceholder' required />
+    );
+  }
+
+  renderEmailField() {
+    return (
+      <Field name="email" component={this.renderEmailFieldComponent} />
+    );
+  }
+
+  renderPrivacyField() {
+    const fieldName = 'privacy';
+    const url = '/privacy';
+    const l10nArgs = JSON.stringify({ url });
+
+    return (
+      <label className={this.makeRevealedClassNames()} htmlFor={fieldName}>
+        <Field name={fieldName} component="input" type="checkbox" required />
+        <span data-l10n-id="emailFormPrivacyNotice" data-l10n-args={l10nArgs}>
+          I'm okay with Mozilla handling by info as explained in
+          <a href={url}>this Privacy Notice</a>.</span>
+      </label>
+    );
+  }
+
+  renderSubmitButton() {
+    if (this.props.submitting) {
+      return (
+        <button disabled={true}
+                data-l10n-id='emailFormSubmitButtonSubmitting'>
+          Submitting...
+        </button>
+      );
+    }
+    return <button data-l10n-id='emailFormSubmitButton'>Sign Up Now</button>;
+  }
+
+  renderDisclaimer() {
+    return (
+      <p className={this.makeRevealedClassNames()}
+         data-l10n-id='emailFormDisclaimer'>
+        We will only send you Test Pilot-related information.
+      </p>
+    );
+  }
+
+  subscribeToBasket(values) {
```

Comment:
Yup, most of the code is lifted from there. As I mentioned in comment 0, I'm trying to appropriately encapsulate all the functionality in the form so it can be reused elsewhere without dependency. I'll likely remove that code when addressing #1551.